utility/src/main/java/org/oppia/util/logging/firebase/LogReportingModule.kt

```diff
@@ -13,18 +14,34 @@ import javax.inject.Singleton
 /** Provides Firebase-specific logging implementations. */
 @Module
 class LogReportingModule {
+
+  @Provides
+  @EnableDataCollection
+  @Singleton
+  fun enableDataCollection(): Boolean {
```

Comment:
I suggest injecting the field in the firebase impl and disabling functionality that way rather than using a stub.

Reply:
I tried that, but then the tests started failing because it had FirebaseAnalytics as one of the parameters which was returning a null value. Nevertheless, will keep this in mind in future :)